ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/snap/SnapServer.java

```diff
@@ -524,7 +525,7 @@ MessageData constructGetTrieNodesResponse(final MessageData message) {
                     var trieNode = optStorage.orElse(Bytes.EMPTY);
                     if (!trieNodes.isEmpty()
                         && (sumListBytes(trieNodes) + trieNode.size() > maxResponseBytes
-                            || stopWatch.getTime()
+                            || stopWatch.getTime(TimeUnit.MILLISECONDS)
```

Comment:
time units passed to method. I think milliseconds is right?

Reply:
yeah milliseconds is the right unit